src/window.ts

```diff
@@ -1,12 +1,16 @@
 import jsdom from 'jsdom';
 
-let namespace: Window & typeof globalThis;
+let namespace: Window & typeof globalThis & {
+    _jsdom?: jsdom.JSDOM;
+};
```

Comment:
I would like to export this complex type as named type, in order to easily re-use it as reference. Something like:

```ts
export type GlobalNamespace = Window & typeof globalThis & {
    _jsdom?: jsdom.JSDOM;
};

let namespace: GlobalNamespace;
```

It can be reused at line 12 and it can be exported as type in the index.ts.

Reply:
Thanks, should be ok now.